database/queries.go

```diff
@@ -0,0 +1,49 @@
+package database
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+)
+
+var (
+	ErrNoSuitableEEFound = errors.New("no suitable key found in database")
+)
+
+// GetLabelOfLatestEE returns the label of the latest end-entity for the specified signer
+// that is no older than a given duration
+func (db *Handler) GetLabelOfLatestEE(signerId string, youngerThan time.Duration) (label, x5u string, err error) {

```

Comment:
I think we could validate in app that labels match a certain format

Reply:
Should we maybe push that up one level and validate that signer IDs match a certain format?

Reply:
Added to main.go, now returns `signer ID \"normandy(@#*71)\" does not match the permitted format \"^[a-zA-Z0-9-_]{1,64}$\""`

Reply:
Nice! :100: